src/eds_files/readme.txt

```diff
@@ -0,0 +1,33 @@
+Where means 'h' for hexidecimal number
```

Comment:
Ik weet niet of het in alle talen zo is, maar in Python schrijf je een hex getal met 0x[nummer]. Dit is verder wel duidelijk, dus je hoeft het niet per se aan te passen, vind ik

Reply:
Nee zo wordt het beschreven in die canopen documenten